src/components/StoreItemCard/index.tsx

```diff
@@ -1,9 +1,28 @@
 import React from 'react';
+import { ReactComponent as Diamond } from '../../assets/icons/Diamond.svg';
+import CatBread from '../../assets/icons/cat_bread.jpg';
 
 import './style.less';
 
 const ItemCard: React.FC = () => {
-  return <div className="item-card">This is a placeholder.</div>;
+  return (
+    <div className="item-card">
+      <div className="inside-rectangle">
+        <div className="rectangle top" />
+        <div className="rectangle bottom" />
+        <div className="image-container">
+          <img className="image" src={CatBread} alt="Lorem Ipsum" />
+        </div>
+      </div>
+      <div className="info">
+        <p className="title">Cat Bread</p>
+        <div className="price">
+          <Diamond />
+          <p className="amount">5,000</p>
+        </div>
+      </div>
+    </div>
+  );
```

Comment:
Why not configure with props?

Reply:
To be fair, I did take this directly from Jennifer's PR, I did not modify anything to maintain relative identity between versions of PR's.

I added properties in 924c19b, though.